wiki/Do_you_really_want_to_ask_peppy/pl.md

```diff
@@ -0,0 +1,15 @@
+# Czy naprawdę chcesz skontaktować się z peppym?
+
+Cześć. Ja ([peppy](/users/2)) rozumiem twoją chęć do prywatnego skontaktowania się ze mną, ale musisz najpierw zrozumieć, że to **nie** jest miejsce, gdzie możesz się mnię pytać o pomoc. Oddzielam pomoc od prywatnych wiadomości. Musisz także zrozumieć, że odpowiadanie na prywatne wiadomości na forum jest dla mnie trudne do nadążenia. W związku z tym możesz użyć jednej z poniższych opcji:
```

Comment:
No elo.

Cześć. Ja ([peppy](/users/2)) rozumiem twoją chęć do prywatnego skontaktowania się ze mną, ale musisz najpierw zrozumieć, że to **nie** jest miejsce, gdzie możesz mnie poprosić o pomoc. Oddzielam pomoc techniczną dla osu! od prywatnych wiadomości, a samo nadążanie za tymi drugimi jest już wystarczająco trudne i męczące. W związku z tym pragnę przyciągnąć twoją uwagę do poniższych opcji. Skorzystaj z tej, która najbardziej odpowiada twojemu problemowi.